utils/src/main/kotlin/Utils.kt

```diff
@@ -391,6 +392,12 @@ fun JsonNode?.textValueOrEmpty(): String = this?.textValue()?.let { it } ?: ""
  */
 fun String.encodeOrUnknown() = fileSystemEncode().takeUnless { it.isBlank() } ?: "unknown"
 
+/*
+ * Convenience function for [JsonNode] that returns the an empty iterator
```

Comment:
Remove "the", also make use of the 120 chars line length limit.

Reply:
Done.